.circleci/config.yml

```diff
@@ -124,6 +124,11 @@ workflows:
             - manylinux-x86_cp37-cp37m
             # python 2.7 test
             - manylinux-x64_cp27-cp27mu_tests-only
+          filters:
+            branches:
+              only: master
+            tags:
+              only: /.*/
```

Comment:
Since we want the `check-dist` to also run on PR, I suggest to use `<<: *no_filters`

Reply:
I don't think you can merge keys? `<<` only works if there are no other keys in the dict.